phpBB/language/en/email/user_remind_username.txt

```diff
@@ -0,0 +1,13 @@
+Subject: {SITENAME} Username reminder
+
+Hello {USERNAME},
+
+Somebody (perhaps you) has requested a reminder of the username for your account at "{SITENAME}". Your Username is: {USERNAME}
+
+You can login to "{SITENAME}"at {U_LOGIN}, or use the 'login' link on the forum home page: .  Note that the username and password are case-sensitive.
```

Comment:
This line has several issues:
* no space between site name and `at`;
* empty homepage link.

Reply:
I have pushed a fix.  

Note that there is intentionally no homepage link, as the text  'or use the 'login' link on the forum home page.'  is an instruction intended for users who prefer not to click links in emails.